site/content/user-guide/command-and-control.md

```diff
@@ -0,0 +1,90 @@
++++
+title = "Command and Control"
+weight = 250
++++
+
+In Hono 0.6 the first implementation of Command and Control is available that enables applications to send a command
+upstream to a device.
```

Comment:
Since we have now descriptions in API, Concepts and User Guide we maybe should link each at the top? WDYT?

Reply:
Good proposal, will include links :+1: